app/views/dashboard/_new_hosts_widget_host_list.html.erb

```diff
@@ -0,0 +1,18 @@
+<table class="table table-striped ellipsis">
```

Comment:
`ellipsis` class should be on the `<td>`, not the table. the table should have `table-fixed` and the `th` elements should set their width.